L/likwid/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "likwid"
+version = v"5.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/RRZE-HPC/likwid/archive/refs/tags/v$(version).tar.gz", "7dda6af722e04a6c40536fc9f89766ce10f595a8569b29e80563767a6a8f940e"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd likwid-5.2.2/
```

Comment:
Just to be future-proof
```suggestion
cd likwid-*
```